pkg/cmd/server/admin/create_mastercerts.go

```diff
@@ -113,6 +118,8 @@ func NewCommandCreateMasterCerts(commandName string, fullName string, out io.Wri
 	flags.StringSliceVar(&options.Hostnames, "hostnames", options.Hostnames, "Every hostname or IP that server certs should be valid for (comma-delimited list)")
 	flags.BoolVar(&options.Overwrite, "overwrite", false, "Overwrite all existing cert/key/config files (WARNING: includes signer/CA)")
 
+	flags.IntVar(&options.ExpireDays, "expire-days", options.ExpireDays, "Validity of the certificates in days (defaults to 2 years). WARNING: extending this above default value is highly discouraged.")
```

Comment:
Ah, good catch! Yes, it doesn't affect CA certificate. At least, I can mention this fact in the description.

@mfojtik I can add dedicated option for CA lifetime or leave it as-is and just mention that CA lifetime will always have default value. WDYT?